apps/api/src/app/events/usecases/message-matcher/message-matcher.usecase.ts

```diff
@@ -27,6 +37,7 @@ import {
   CreateExecutionDetailsCommand,
   DetailEnum,
 } from '../../../execution-details/usecases/create-execution-details/create-execution-details.command';
+import { EmailEventStatusEnum } from '@novu/stateless';
```

Comment:
Should this enum be moved to `@novu/shared`?

Reply:
Yes when we are getting rid of the stateless this should end up in shared. Right now all providers are already dependant on this package so would like to not touch it in this PR.

Reply:
Let's leave it for tech debt. 👍🏻